pravega/pravega-samples
# Issue 113: Problems in anomaly detection application setup

**Change log description**
This PR fixes problems during the setup of `anomaly-detection` application. First, it fixes the `build.gradle` file of `anomaly-detection`, as it has been noticed that it was not correctly creating the executable script to run the application. Moreover, the application was stuck during initialization due to incorrect logging configuration. This PR also changes the logging approach used by `anomaly-detection` to `logback`, which is the scheme used by the rest of sample applications.

**Purpose of the change**
Fixes #113.

**What the code does**
The PR adds in the `gradle.build` new dependencies (`logback`, `lombok`), deletes the old one used for logging and adds the logic to correctly create the execution script. In terms of logging, the `log4j.properties` has been replaced by `logback.xml` and classes using logging now are annotated with `Self4j`. Finally, this PR moves the configuration files to `resources` folder, as in the rest of applications and adds minor changes in the README file. 

**How to verify it**
Clone the PR patch, start Pravega standalone and execute `./gradlew installDist`. Then, go to the `anomaly-detection` folder and execute the `bin/anomaly-detection` script as specified in the instructions. The application should run as expected.
